### PR TITLE
issue #539 - support redirect of partition queue

### PR DESCRIPTION
**Pull request by dwieland**
#539

### CODE REVIEW

**Comment by edenhill** [comment]
Some remaining things:
 * You will need to add the symbol to tests/0006-symbol.c  (with all arguments being 0 or NULL)
 * Add a C++ interface (optional)
 * Add a test case (optional)

**Comment by dwieland** [comment]
I will take a look at the tests later on.

File: src-cpp/KafkaConsumerImpl.cpp [hunk 1]
```diff
@@ -172,6 +183,24 @@ RdKafka::KafkaConsumerImpl::assign (const std::vector<TopicPartition*> &partitio
 
 
 RdKafka::ErrorCode
+RdKafka::KafkaConsumerImpl::redirectPartitionQueue (const TopicPartition *partition,
```
Comment (edenhill) [inline src-cpp/KafkaConsumerImpl.cpp]: Lets use the same name as in the C code as to make documentation references agnostic. (e.g. setPartitionQueue)

**Comment by edenhill** [comment]
work is being done in #953

**Pull request closed without merging**